src/main/java/com/example/bookstoreapp/models/ShoppingOrder.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bookstoreapp.models;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ShoppingOrder {
+  private Long id;
+  private Float totalAmount;
+  private int totalItemCount;
+  private List<OrderItem> orderItems;
+  private String address;
```

Comment:
Why we have address here

Reply:
Because each order should be shipped to some address.